### DISCUSSION TITLE
How do I start writing my own patches?

### DISCUSSION THREAD

**Discussion by muradbu** [ReVanced/revanced-discussions#961] [Question] [answered]
Is there a general guide or other resources on how to start writing your own patches? It seems like a valuable skill to learn for myself, as well as for the community.

**Comment by reisxd** [comment, marked as answer]
https://github.com/revanced/revanced-patches/tree/feat/docs/docs

You can also check other patches.

**Reply by muradbu** [reply]
Thanks, I wish this was easier to find.
